src/docs/patterns/appbar.mdx

```diff
@@ -1,12 +1,10 @@
 import { DemoCard, ImageGrid, MaterialDesignDescription, Divider } from '../../app/components';
 import Banner from '../../app/assets/design-patterns/app-bar/appbar-banner.png';
-import Image1 from '../../app/assets/design-patterns/app-bar/collapsible-appbar/collapsible-appbar-expanded.png';
-import Image2 from '../../app/assets/design-patterns/app-bar/collapsible-appbar/collapsible-appbar-collapsed.png';
 import Collapsible from '../../app/assets/design-patterns/app-bar/collapsible-appbar/collapsible.gif';
+import Search from '../../app/assets/design-patterns/app-bar/search-bar/search.gif';
```

Comment:
(1) I think the search bar should slide in and cover the app bar, rather than pushing it away
(2) I realized that doc-it's search bar does not match exactly with this screenshot... We might need to update one of it

Reply:
Updating doc-it will be the faster way since it's already implemented like this in the RN component library

Reply:
...layout wise. It doesn't do the slide.

Reply:
We can tell people it's a slide that occurs in 0ms